src/pages/AddPurchase/AddPurchase.js

```diff
@@ -0,0 +1,133 @@
+import FoodList from '../../components/FoodList/FoodList'
```

Comment:
You can improve the order of your imports: First add external imports (e.g. useState, useEffect and styled components) after that group your components and after that the services and lib